.github/workflows/build-fleetd-base-pkg.yml

```diff
@@ -0,0 +1,121 @@
+# This workflow can be used to build a fleetd-base.pkg package
+# that can be hosted on a local server to test ADE workflows.
+#
+# Output is the fleetd-base.pkg itself and the corresponding fleetd-base-manifest.plist.
+name: Build and codesign fleetd-base.pkg
+
+on:
+  workflow_dispatch: # allow manual action
```

Comment:
could we add optional inputs for the `--update-url` and `--update-roots` flag in case we want to use a local TUF server?

Reply:
I would not add those to be on safe-side. (Otherwise anybody could generate fleetd-base.pkg signed by Fleet but with components built who knows where/how.)

Let me know if that makes sense.

Reply:
Ah good point! 

I think we should add this as-is, since it's a big step forward and will enable testing certain flows!

We can brainstorm internally some ways of allowing custom TUF setups in the future perhaps.